text/0000-ton-connect.md

```diff
@@ -0,0 +1,121 @@
+- **TEP**: [0](https://github.com/ton-blockchain/TEPs/pull/0)
+- **title**: TON Connect
+- **status**: Active
+- **type**: Core
+- **authors**: @oleganza, @siandreev, subden, brainpicture, @sorokin0andrey, abogoslavskiy, @MariaBit, Olga May, Aleksei Mazelyuk, tonrostislav, KuznetsovNikita.
+- **created**: 20.10.2022
+- **replaces**: -
+- **replaced by**: -
+
+# Summary
+
+TON Connect is a unified protocol for communication between TON wallets and TON apps.
+
+# Motivation
+
+The Open Network needs a **unified** protocol for communication between TON wallets and TON (d)apps to achieve following goals:
+
+1. Any TON app (web / desktop / mobile / etc) can be operated by any wallet (mobile / desktop / web / browser extension / dapp browser / hardware / etc).
+
+
+2. Users get a familiar and friendly experience across all TON apps.
+
+# Guide
+
+Apps provide the UI to an infinite range of functionality in TON based on smart contracts, but do not have immediate access to users’ funds. Therefore they are often called decentralized apps or “dapps”.
+
+Wallets provide the UI to approving transactions and hold users’ cryptographic keys securely on their personal devices.
+
+This separation of concerns enables rapid innovation and high level of security for the users: wallets do not need to build walled-garden ecosystems themselves, while the apps do not need to take the risk holding end users’ accounts.
+
+TON Connect is a bridge that crosses this conceptual gap.
+
+TON Connect, in addition to the transport layer of communication between the wallet and the app, provides methods of authorization, sending transactions to the network, interaction with smart contracts, etc.
```

Comment:
Which methods of authorization are implied here?
Usually local authorization is done in wallet itself (fingerprint, pin, password).

Is it supposed to be something other than just private key (or complete seed phrase) + local authorization?

E.g. some types of 2FA to sacrifice some decentralization in favor of security?

Reply:
Authorization in this context means proving to the app backed that user really owns the wallet.
It is also shown in docs: https://ton.org/docs/develop/dapps/ton-connect/integration#logging-out-and-requesting-tonproof

Reply:
Thanks a lot!
Need to go through this thoroughly.